core/sys/posix/sys_stat.odin

```diff
@@ -427,6 +427,9 @@ when ODIN_OS == .Darwin {
 	UTIME_NOW  :: -2
 	UTIME_OMIT :: -1
 
+} else when ODIN_OS == .Linux {
+	ino_t   :: distinct c.unit64_t
+	ino_t32 :: distinct c.unit32_t
```

Comment:
Typos here. `unit` -> `uint`, and I'd also prefer `u64` and `u32` instead too in general

Reply:
@gingerBill Thanks for spotting this.
I was following the style of the package with the types but would love to use the `builtin` types directly.
How about making this same change on the entire file and places of the package where it makes sense?

Reply:
Honestly, it doesn't really matter which you use. It's more of an aesthetic judgment as to which is better. As long as you are consistent. with the `core` packages, I'd probably prefer `u64` over `c.uint64_t`, but if you have one of the more exotic types, then sure use the `c.*_t` equivalent.